src/app/core/components/public-settings/public-settings.component.html

```diff
@@ -32,4 +32,13 @@
     Display a button that allows viewers of your published items to download a
     copy of the source file to their device.
   </small>
+  <div class="settings-item">
+   <label>Archive Type</label>
+   <select [(ngModel)]="archiveType" name="archiveType" id="archiveType" (ngModelChange)="onArchiveTypeChange()" [disabled]="archive.type === 'type.archive.nonprofit'" class="custom-select">
```

Comment:
I know this is a little tricky since we don't have a specific design, but I don't think this version will work. It seems too confusing for a user looking at settings for a nonprofit archive -- "why is the type blank?" they'd be wondering. My idea would be to not show the dropdown at all and just show the word "Nonprofit" in that case. Open to other ideas!

Reply:
I think that is a great idea! The user might see a disabled dropdown and think there is something wrong with their account. I will change it now :)